flake.nix

```diff
@@ -5,6 +5,10 @@
   nixConfig = {
     trusted-substituters =
       [ "https://cache.nixos.org/" "https://deku.cachix.org" ];
+    trusted-public-keys = [
```

Comment:
I think we can cut this PR down to this change  and then try to get #576 merged instead